jekyll/_cci2/gitlab-integration.adoc

```diff
@@ -20,39 +20,47 @@ This document walks you through integrating a GitLab project with CircleCI. The
 [#sign-up]
 == Sign up
 
-GitLab support is open to both new and existing users of CircleCI. Follow the steps on the xref:first-steps#gitlab-signup[Sign up and try CircleCI] page to:
+GitLab support is open to both new and existing users of CircleCI. Follow the steps on the xref:first-steps#gitlab-signup[Sign up and try CircleCI] page to connect your GitLab account to create your organization.
 
-* Connect your GitLab account to create your organization.
-* Create a project and connect a repository from your GitLab account.
+NOTE: You will need API access and write permissions on the repository you want to set up. Within GitLab, this is the “maintainer” role or higher.
 
-When you connect a repo with your CircleCI project, behind the scenes, CircleCI is registering a webhook within your GitLab repository. You may verify this once you have successfully created the project by navigating to your repository's **Settings > Webhooks** page.
+When you create a new organization and connect your GitLab account, you will be prompted to create a new project from a repository. You will also have the following options to set up a CircleCI configuration for your project:
```

Comment:
You will have these options (Fastest, Faster and Fast) if the repository selected doesn't already have a circleci config file

Reply:
@kbatuigas this sounds like it differs to how things are in the current GH/BB flow then, because currently setting up a project where there is already a config file in the repo you still get presented with the Fastest, Faster and Fast options 🤔

Reply:
@rosieyohannan Honestly I am confused about some of the flow in the current GH/BB environment. Why do we show fast/faster/fastest for a project with a circleci config already? The point of showing these options is for setting up the config file. It feels redundant but I guess this is a problem to solve with Product. I can raise this with them.

Reply:
@adewunmiadesokan yeah this is definitely still the case - just confirmed with a BB repo I have that already contains a config

![image](https://user-images.githubusercontent.com/36839689/222511845-00a0ea53-4089-432a-89ae-05ab7df92432.png)